components/hab/src/cli.rs

```diff
@@ -902,6 +902,12 @@ pub fn sub_sup_run() -> App<'static, 'static> {
             itself")
         (@arg EVENTS: --events -n +takes_value {valid_service_group} "Name of the service \
             group running a Habitat EventSrv to forward Supervisor and service event data to")
+        (@arg KEYFILE: --key +takes_value
+            "Read private key from KEYFILE.  This should be a RSA private key or PKCS8-encoded \
+             private key, in PEM format.")
+        (@arg CERTFILE: --certs +takes_value
+            "Read server certificates from CERTFILE. This should contain PEM-format certificates \
+             in the right order (the first certificate should certify KEYFILE, the last should be a root CA)")
```

Comment:
The documentation should indicate that these are for the HTTP gateway, and not anything else.

Reply:
Good call, will update